core/capabilities/remote/target/client.go

```diff
@@ -146,6 +146,8 @@ func (c *client) Receive(msg *types.MessageBody) {
 		return
 	}
 
+	// note this go function is removed in PR https://github.com/smartcontractkit/chainlink/pull/13524 so passing
```

Comment:
Remove these comments before merging.

Reply:
will do, going to leave comments in place until I have the second review in place